src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+           --url 'https://cockroachlabs.cloud/api/v1/folders/root/contents' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace `<api-key>` with your service account's API key.
+
+1. To list the clusters and folders contained within another folder, specify its ID instead of `root`:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+           --url 'https://cockroachlabs.cloud/api/v1/folders/<folder_id>/contents' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace:
+    - `<folder-id>`: The ID of a folder.
+    - `<api-key>`: Your service account's API key.
+
+{{site.data.alerts.callout_info}}
+In the API, each folder is referred to by its ID, regardless of where it is located in the hierarchy.
+{{site.data.alerts.end}}
+
+## Create a cluster in a folder
+
+Your service account must have the following roles on the organization or the folder:
+
+- [Cluster Administrator](authorization.html#cluster-administrator) or [Cluster Creator](authorization.html#cluster-creator)
+
+To create a cluster in a folder, set the `parent_id` field to the ID of the folder.
+
+{{site.data.alerts.callout_info}}
+If you do not specify a `parent_id`, or if you set `parent_id` to `root`, the cluster is created at the level of the organization.
+{{site.data.alerts.end}}
+
+For example, to create a {{ site.data.products.serverless }} deployed on GCP in a folder:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+       --url 'https://cockroachlabs.cloud/api/v1/clusters' \
+       --header 'Authorization: Bearer <api-key>' \
+       --header 'content-type: application/json' \
+       --data '{"name":"<cluster_name>","provider":"GCP","spec":{"serverless":{"regions":["us-central1"],"spend_limit":0}, "parent_id":"<parent_id>"}}'
+~~~
+
+Replace:
+
+- `<cluster_name>`: A name for the cluster. Replace `<parent_id>` with the ID of the folder.
+- `<api-key>`: Your service account's API key.
+- `<parent-id>`: The ID of a folder.
+
+## Move a cluster into or out of a folder
+
+Your service account must have the following roles on the organization or on both the source and destination locations:
+
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
+
+{{site.data.alerts.callout_info}}
+Folder Movers can move clusters within the folder hierarchy even if they do not have a role that allows them to connect to the cluster, such as Cluster Creator or Cluster Operator.
+{{site.data.alerts.end}}
+
+1. To move a cluster from the organization level into a folder, or to move it from one folder to another:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request PATCH \
+           --url 'https://cockroachlabs.cloud/api/v1/clusters/136cf380-91b1-48ff-9393-966ab10d946f' \
```

Comment:
nit: `<cluster_id>` instead of a real ID

Reply:
Done